src/System.Security.Principal.Windows/tests/System.Security.Principal.Windows.Tests.csproj

```diff
@@ -20,11 +20,5 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
This should remain an pkgproj reference to the library under test. Whey did you need to remove it?